test/unit/index.test.ts

```diff
@@ -153,12 +149,4 @@ describe('mongodb entrypoint', () => {
   it('exports only the expected keys', () => {
     expect(setDifference(Object.keys(mongodb), EXPECTED_EXPORTS)).to.be.empty;
   });
-
-  it('should export keys added by ts-node as undefined', () => {
```

Comment:
Did this test start failing? Did we update something about ts-node in just this PR?

Reply:
- https://github.com/mongodb/node-mongodb-native/pull/4292#discussion_r1826104349
- https://github.com/mongodb/node-mongodb-native/pull/4292#pullrequestreview-2408002618